examples/query/account_balance_query.py

```
@@ -30,73 +26,56 @@
         )
 
-    operator_id = AccountId.from_string(operator_id_str)
-    operator_key = PrivateKey.from_string(operator_key_str)
-    client.set_operator(operator_id, operator_key)
-
-    print(f"Client set up with operator id {client.operator_account_id}")
+    print(f"Client set up with operator id {operator_id}")
     return client, operator_id, operator_key
 
 
 def create_account(client, operator_key, initial_balance=Hbar(10)):
```

Ruff (0.14.13)

Do not perform function call `Hbar` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)